### PR TITLE
Composer improvements and migrate for 3.0

### PR DESCRIPTION
**Pull request by alexkvak**

### CODE REVIEW

**Comment by Soullivaneuh** [comment]
> Could you please update your PR and commit to have something like this one ? https://github.com/sonata-project/SonataCoreBundle/pull/240

**Comment by alexkvak** [comment]
@Soullivaneuh Yes, I can, but I don't understand, what version choices I need to use

**Comment by Soullivaneuh** [comment]
@alexkvak Branch alias should be `3.x-dev` and composer should follow the new format: https://github.com/sonata-project/SonataCoreBundle/pull/240/files

**Comment by alexkvak** [comment]
@Soullivaneuh under new format you mean Caret ^ instead of Tilde ~ ?

**Comment by Soullivaneuh** [comment]
Yes. And ` || ` instead of `|`.

Sullivan SENECHAL

File: composer.json [hunk 1]
```diff
@@ -17,25 +17,25 @@
...
+        "sonata-project/datagrid-bundle": "^2.2",
+        "cocur/slugify": "^1.0"
     },
     "require-dev": {
-        "sonata-project/doctrine-orm-admin-bundle": "~2.2",
-        "symfony/phpunit-bridge": "~2.7|~3.0",
-        "friendsofsymfony/rest-bundle": "~1.1",
-        "jms/serializer-bundle": "~0.11|~1.0",
-        "nelmio/api-doc-bundle": "~2.4"
+        "sonata-project/doctrine-orm-admin-bundle": "^2.2",
```
Comment (jordisala1991) [inline composer.json]: This should be ^3.0 right?

Comment (Soullivaneuh) [inline composer.json]: Yes.

**Comment by Soullivaneuh** [comment]
Done in https://github.com/sonata-project/SonataPageBundle/commit/45698823a8a092dd469e4edaabf1ba0d2c7ebd5f.

**Pull request closed without merging**